package.json

```diff
@@ -1,12 +1,23 @@
 {
-  "name": "react-boilerplate",
+  "name": "cyber-pizza",
   "version": "1.0.0",
-  "description": "",
+  "description": "Pizza-shop site",
   "main": "index.js",
   "scripts": {
     "start": "node server.js",
     "build": "webpack --mode production",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "server:watch": "nodemon server.js",
+    "client:watch": "webpack serve",
+    "lint": "eslint \"{,!(node_modules|dist)/**/}*.+(js|jsx)\" --fix",
+    "format": "prettier --write \"{,!(node_modules|dist)/**/}*.+(js|jsx|json|css|md)\"",
+    "prepare": "husky install",
+    "deploy": "npm install && git add . && git commit -m \"deploy on heroku\" && git push heroku -f production:master"
+  },
+  "prettier": {
+    "singleQuote": true,
+    "printWidth": 2,
```

Comment:
`printWidth` is length of characters per line, which shouldn't be wrapped, please set this param to 100, and add set `tabWidth` to 2